dotcom-rendering/src/components/InteractiveBlockComponent.importable.tsx

```diff
@@ -346,23 +346,27 @@ export const InteractiveBlockComponent = ({
 			}
 
 			// Datawrapper-specific fix to suppress scrollbars appearing
-			if (url.includes('datawrapper')) {
-				iframe.scrolling = 'no';
-				// Turn off dark mode for Datawrapper embeds on web
-				// This should be removed if/when dark mode is implements on the website
-				if (
-					!document.querySelector('.ios') &&
-					!document.querySelector('.android')
-				) {
-					const prefersDarkScheme = window.matchMedia(
-						'(prefers-color-scheme: dark)',
-					).matches;
-					const darkMode = darkModeAvailable && prefersDarkScheme;
-					if (!darkMode) {
-						iframe.src +=
-							(iframe.src.includes('?') ? '&' : '?') +
-							'dark=false';
-					}
+			if (url.includes('datawrapper')) iframe.scrolling = 'no';
```

Comment:
I've noticed there's an `isDatawrapperGraphic` boolean flag defined around line 400ish. It's not an elegant function (I much prefer `includes()`) but it may make sense to hoist the const up the file so we only check once and then use the flag rather than repeat the check? How about something like:
```
const isDatawrapperGraphic = url.includes('interactive.guim.co.uk/datawrapper');
```